src/uproot_browser/plot_view.py

```diff
@@ -63,14 +64,33 @@ async def update(self) -> None:
         self.refresh()
 
     def render(self) -> rich.console.RenderableType:
-        if self.plot is None or self.plot is EMPTY:
```

Comment:
Maybe the best way to handle errors would be to put a try/except block, then put the exception object in as `self.plot`. Then if self.plot isinstance Exception, that gets displayed (probably a rich method somewhere that renders exceptions, should be in a scroll view). You could also set it to a Text string, and if it's Text, that's a "known error" and that gets displayed (like "not a known plotable object" or something, maybe eventually some details like "Show" in ROOT would show.

Reply:
Dynamically adding the scroll view might be a problem, if a scroll view is not a rich.console.RenderableType. That can be a second step, or maybe we can manage to classify all the likely errors.